docker/README.md

```diff
@@ -0,0 +1,149 @@
+# Building KubeStellar deployment as a container
+
+Table of contents:
+- [KubeStellar in a container](#kubestellar-in-a-container)
+  - [Install the pre-requisites](#install-the-pre-requisites)
+  - [Build a local container image](#build-a-local-container-image)
+  - [Build and push a multi-architecture image](#build-and-push-a-multi-architecture-image)
+  - [Run the container and access KubeStellar from the host OS](#run-the-container-and-access-kubestellar-from-the-host-os)
+  - [Checking the container logs](#checking-the-container-logs)
+  - [Login into the running container to use **KubeStellar** from inside the container](#login-into-the-running-container-to-use-kubestellar-from-inside-the-container)
+  - [Teardown and cleanup](#teardown-and-cleanup)
+
+## Install the pre-requisites
+
+Install `nake`:
+
+```bash
+sudo apt install -y make
+```
+
+Install `docker`:
+
+```bash
+sudo mkdir -p /etc/apt/keyrings
+curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo gpg --dearmor -o /etc/apt/keyrings/docker.gpg
+echo "deb [arch=$(dpkg --print-architecture) signed-by=/etc/apt/keyrings/docker.gpg] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable" | sudo tee /etc/apt/sources.list.d/docker.list > /dev/null
+sudo apt update
+sudo apt install -y docker-ce docker-ce-cli containerd.io docker-compose-plugin
+sudo usermod -aG docker $USER
+```

```

Comment:
I am not a fan of maintaining a copy of someone else's instructions.  I think a pointer to them would suffice.

Reply:
granted, this is a convenience

Reply:
we can evolve this in the future to be a pointer to external doc.  Let's get this out first and rev the documentation later.  I am ok with this being a copy for now since it is specific to Debian.